functional/README.md

```diff
@@ -10,6 +10,23 @@ Since the tests utilize [`systemd-nspawn`][systemd-nspawn], this needs to be inv
 
 If the tests are aborted partway through, it's currently possible for them to leave residual state as a result of the `systemd-nspawn` operations. This can be cleaned up using the `clean.sh` script.
 
+### Using go flags in functional tests
+
+`fleet/functional` scripts forwards all arguments directly to the `go` binary. This allows you to pass `-run regexp` argument to the `./test` script and run specific test functions. The example below shows how to run only `TestSchedule*` test functions:
+
+```sh
+$ sudo fleet/functional/test -run TestSchedule*
+```
+
+The test functions list could be printed using `grep` command:
+
+```sh
+$ cd fleet/functional
+$ grep -r 'func Test' .
+```
+
+You can find detailed description of the available test flags in the [Go documentation][golang-test-flags].
```

Comment:
Argh, I'm sorry, this is totally my bad (!!), but it should be **a** detailed description.